### ISSUE TITLE
Count error

### ISSUE THREAD

**Issue by rickdonato** [netenglabs/suzieq#636]
**Describe the bug**
Error when performing a count.

**To Reproduce**
```
rick> inventory unique columns="serial" count="True"
Error running command: 'numRows'
------------------------------------------------------------
Traceback (most recent call last):
  File "/home/rick/development/course-dev/suzieq-bootcamp/venv/venv-sq-bootcamp/lib/python3.8/site-packages/nubia/internal/cmdbase.py", line 408, in run_interactive
    ret = fn(**args_dict)
  File "/home/rick/development/course-dev/suzieq-bootcamp/venv/venv-sq-bootcamp/lib/python3.8/site-packages/suzieq/cli/sqcmds/command.py", line 211, in unique
    df.sort_values(by=['numRows', df.columns[0]]),
  File "/home/rick/development/course-dev/suzieq-bootcamp/venv/venv-sq-bootcamp/lib/python3.8/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/rick/development/course-dev/suzieq-bootcamp/venv/venv-sq-bootcamp/lib/python3.8/site-packages/pandas/core/frame.py", line 6242, in sort_values
    keys = [self._get_label_or_level_values(x, axis=axis) for x in by]
  File "/home/rick/development/course-dev/suzieq-bootcamp/venv/venv-sq-bootcamp/lib/python3.8/site-packages/pandas/core/frame.py", line 6242, in <listcomp>
    keys = [self._get_label_or_level_values(x, axis=axis) for x in by]
  File "/home/rick/development/course-dev/suzieq-bootcamp/venv/venv-sq-bootcamp/lib/python3.8/site-packages/pandas/core/generic.py", line 1779, in _get_label_or_level_values
    raise KeyError(key)
KeyError: 'numRows'
------------------------------------------------------------
rick> 
```

**Attach**
TBC

 **Expected behavior**
TBC

**Screenshots**
TBC

**Additional context**
Environment from which this is occurring in:
* remote cli is being used
* correct port setting applied via "set rest-server-port=443"
* can successfully query other tables. For example "device show"

**Local DB works, but remote dosn't**

**Comment by rickdonato** [comment]
Yep ... 
```
rick> bgp show                                                                                                                                     Error: 1
   namespace     hostname      vrf         peer peerHostname        state  ...    asn peerAsn  pfxRx  pfxTx  numChanges                         estdTime
0     nebula    leaf1-ios  default  11.11.11.11               Established  ...  65011   65000      0      0           1 2022-02-25 13:34:48.976000+00:00
1     nebula    leaf1-ios  default  22.22.22.22               Established  ...  65011   65000      0      0           3 2022-02-26 13:34:48.976000+00:00
2     nebula    leaf2-ios  default  11.11.11.11               Established  ...  65011   65000      0      0           1 2022-03-15 11:34:47.246000+00:00
3     nebula    leaf2-ios  default  22.22.22.22               Established  ...  65011   65000      0      0           1 2022-03-15 11:34:47.246000+00:00
4     nebula    leaf3-qfx  default  11.11.11.11               Established  ...  65012   65000      0      0           0 2022-01-12 22:34:59.661000+00:00
5     nebula    leaf3-qfx  default  22.22.22.22               Established  ...  65012   65000      0      0           2 2022-02-21 15:51:36.661000+00:00
6     nebula    leaf4-qfx  default  11.11.11.11               Established  ...  65012   65000      0      0           0 2022-01-12 22:34:58.859000+00:00
7     nebula    leaf4-qfx  default  22.22.22.22               Established  ...  65012   65000      0      0           2 2022-02-21 15:51:18.859000+00:00
8     nebula    leaf5-eos  default  11.11.11.11               Established  ...  65013   65000      0      0           1 2022-01-12 22:34:46.996000+00:00
9     nebula    leaf5-eos  default  11.11.11.11               Established  ...  65013   65000      0      0           1 2022-01-12 22:34:46.996000+00:00
10    nebula    leaf5-eos  default  22.22.22.22               Established  ...  65013   65000      0      0           2 2022-02-21 16:34:46.996000+00:00
11    nebula    leaf5-eos  default  22.22.22.22               Established  ...  65013   65000      0      0           2 2022-02-21 16:34:46.996000+00:00
12    nebula    leaf6-eos  default  11.11.11.11               Established  ...  65013   65000      0      0           1 2022-01-12 23:34:46.897000+00:00
13    nebula    leaf6-eos  default  11.11.11.11               Established  ...  65013   65000      0      0           1 2022-01-12 23:34:46.897000+00:00
14    nebula    leaf6-eos  default  22.22.22.22               Established  ...  65013   65000      0      0           2 2022-02-21 16:34:46.897000+00:00
15    nebula    leaf6-eos  default  22.22.22.22               Established  ...  65013   65000      0      0           2 2022-02-21 16:34:46.897000+00:00
16    nebula  spine2-nxos  default      1.1.1.1                   NotEstd  ...  65000   65011      0      0           2 2022-02-21 05:19:43.078000+00:00
17    nebula  spine2-nxos  default      2.2.2.2                   NotEstd  ...  65000   65011      0      0           2 2022-02-21 05:20:13.078000+00:00
18    nebula  spine2-nxos  default      3.3.3.3                   NotEstd  ...  65000   65012      0      0           4 2022-02-21 05:19:10.078000+00:00
19    nebula  spine2-nxos  default      4.4.4.4                   NotEstd  ...  65000   65012      0      0           4 2022-02-21 05:18:58.078000+00:00
20    nebula  spine2-nxos  default      5.5.5.5                   NotEstd  ...  65000   65013      0      0           2 2022-02-21 05:20:41.078000+00:00
21    nebula  spine2-nxos  default      6.6.6.6                   NotEstd  ...  65000   65013      0      0           2 2022-02-21 05:19:43.078000+00:00
```

**Comment by ddutt** [comment]
Sorry, in this case, can you do inventory show?

**Comment by rickdonato** [comment]
This is what I get:
```
rick> inventory show
    namespace     hostname          name                                model   status vendor      type version
0      nebula  spine1-nxos   Ethernet1/1                                        absent             xcvr        
1      nebula  spine1-nxos   Ethernet1/2                                        absent             xcvr        
2      nebula  spine1-nxos   Ethernet1/3                                        absent             xcvr        
3      nebula  spine1-nxos   Ethernet1/4                                        absent             xcvr        
4      nebula  spine1-nxos   Ethernet1/5                                        absent             xcvr        
5      nebula  spine1-nxos   Ethernet1/6                                        absent             xcvr        
6      nebula  spine1-nxos   Ethernet1/7                                        absent             xcvr        
7      nebula  spine1-nxos   Ethernet1/8                                        absent             xcvr        
8      nebula  spine1-nxos   Ethernet1/9                                        absent             xcvr        
9      nebula  spine1-nxos  Ethernet1/10                                        absent             xcvr        
10     nebula  spine1-nxos  Ethernet1/11                                        absent             xcvr        
11     nebula  spine1-nxos  Ethernet1/12                                        absent             xcvr        
12     nebula  spine1-nxos  Ethernet1/13                                        absent             xcvr        
13     nebula  spine1-nxos  Ethernet1/14                                        absent             xcvr        
14     nebula  spine1-nxos  Ethernet1/15                                        absent             xcvr        
15     nebula  spine1-nxos  Ethernet1/16                                        absent             xcvr        
16     nebula  spine1-nxos  Ethernet1/17                                        absent             xcvr        
17     nebula  spine1-nxos  Ethernet1/18                                        absent             xcvr        
18     nebula  spine1-nxos  Ethernet1/19                                        absent             xcvr        
19     nebula  spine1-nxos  Ethernet1/20                                        absent             xcvr        
20     nebula  spine1-nxos  Ethernet1/21                                        absent             xcvr        
21     nebula  spine1-nxos  Ethernet1/22                                        absent             xcvr        
22     nebula  spine1-nxos  Ethernet1/23                                        absent             xcvr        
23     nebula  spine1-nxos  Ethernet1/24                                        absent             xcvr        
24     nebula  spine1-nxos  Ethernet1/25                                        absent             xcvr        
25     nebula  spine1-nxos  Ethernet1/26                                        absent             xcvr        
26     nebula  spine1-nxos  Ethernet1/27                                        absent             xcvr        
27     nebula  spine1-nxos  Ethernet1/28                                        absent             xcvr        
28     nebula  spine1-nxos  Ethernet1/29                                        absent             xcvr        
29     nebula  spine1-nxos  Ethernet1/30                                        absent             xcvr        
30     nebula  spine1-nxos  Ethernet1/31                                        absent             xcvr        
31     nebula  spine1-nxos  Ethernet1/32                                        absent             xcvr        
32     nebula  spine1-nxos  Ethernet1/33                                        absent             xcvr        
33     nebula  spine1-nxos  Ethernet1/34                                        absent             xcvr        
34     nebula  spine1-nxos  Ethernet1/35                                        absent             xcvr        
35     nebula  spine1-nxos  Ethernet1/36                                        absent             xcvr        
36     nebula  spine1-nxos  Ethernet1/37                                        absent             xcvr        
37     nebula  spine1-nxos  Ethernet1/38                                        absent             xcvr        
38     nebula  spine1-nxos  Ethernet1/39                                        absent             xcvr        
39     nebula  spine1-nxos  Ethernet1/40                                        absent             xcvr        
40     nebula  spine1-nxos  Ethernet1/41                                        absent             xcvr        
41     nebula  spine1-nxos  Ethernet1/42                                        absent             xcvr        
42     nebula  spine1-nxos  Ethernet1/43                                        absent             xcvr        
43     nebula  spine1-nxos  Ethernet1/44                                        absent             xcvr        
44     nebula  spine1-nxos  Ethernet1/45                                        absent             xcvr        
45     nebula  spine1-nxos  Ethernet1/46                                        absent             xcvr        
46     nebula  spine1-nxos  Ethernet1/47                                        absent             xcvr        
47     nebula  spine1-nxos  Ethernet1/48                                        absent             xcvr        
48     nebula  spine1-nxos  Ethernet1/49                                        absent             xcvr        
49     nebula  spine1-nxos  Ethernet1/50                                        absent             xcvr        
50     nebula  spine1-nxos  Ethernet1/51                                        absent             xcvr        
51     nebula  spine1-nxos  Ethernet1/52                                        absent             xcvr        
52     nebula  spine1-nxos  Ethernet1/53                                        absent             xcvr        
53     nebula  spine1-nxos  Ethernet1/54                                        absent             xcvr        
54     nebula  spine1-nxos  Ethernet1/55                                        absent             xcvr        
55     nebula  spine1-nxos  Ethernet1/56                                        absent             xcvr        
56     nebula  spine1-nxos  Ethernet1/57                                        absent             xcvr        
57     nebula  spine1-nxos  Ethernet1/58                                        absent             xcvr        
58     nebula  spine1-nxos  Ethernet1/59                                        absent             xcvr        
59     nebula  spine1-nxos  Ethernet1/60                                        absent             xcvr        
60     nebula  spine1-nxos  Ethernet1/61                                        absent             xcvr        
61     nebula  spine1-nxos  Ethernet1/62                                        absent             xcvr        
62     nebula  spine1-nxos  Ethernet1/63                                        absent             xcvr        
63     nebula  spine1-nxos  Ethernet1/64                                        absent             xcvr        
64     nebula  spine1-nxos       chassis                     Nexus9000 C9300v  present          chassis        
65     nebula  spine1-nxos    linecard-1  Nexus 9000v 64 port Ethernet Module  present         linecard        
66     nebula  spine1-nxos   linecard-27                    Supervisor Module  present         linecard        
67     nebula  spine2-nxos   Ethernet1/1                                        absent             xcvr        
68     nebula  spine2-nxos   Ethernet1/2                                        absent             xcvr        
69     nebula  spine2-nxos   Ethernet1/3                                        absent             xcvr        
70     nebula  spine2-nxos   Ethernet1/4                                        absent             xcvr        
71     nebula  spine2-nxos   Ethernet1/5                                        absent             xcvr        
72     nebula  spine2-nxos   Ethernet1/6                                        absent             xcvr        
73     nebula  spine2-nxos   Ethernet1/7                                        absent             xcvr        
74     nebula  spine2-nxos   Ethernet1/8                                        absent             xcvr        
75     nebula  spine2-nxos   Ethernet1/9                                        absent             xcvr        
76     nebula  spine2-nxos  Ethernet1/10                                        absent             xcvr        
77     nebula  spine2-nxos  Ethernet1/11                                        absent             xcvr        
78     nebula  spine2-nxos  Ethernet1/12                                        absent             xcvr        
79     nebula  spine2-nxos  Ethernet1/13                                        absent             xcvr        
80     nebula  spine2-nxos  Ethernet1/14                                        absent             xcvr        
81     nebula  spine2-nxos  Ethernet1/15                                        absent             xcvr        
82     nebula  spine2-nxos  Ethernet1/16                                        absent             xcvr        
83     nebula  spine2-nxos  Ethernet1/17                                        absent             xcvr        
84     nebula  spine2-nxos  Ethernet1/18                                        absent             xcvr        
85     nebula  spine2-nxos  Ethernet1/19                                        absent             xcvr        
86     nebula  spine2-nxos  Ethernet1/20                                        absent             xcvr        
87     nebula  spine2-nxos  Ethernet1/21                                        absent             xcvr        
88     nebula  spine2-nxos  Ethernet1/22                                        absent             xcvr        
89     nebula  spine2-nxos  Ethernet1/23                                        absent             xcvr        
90     nebula  spine2-nxos  Ethernet1/24                                        absent             xcvr        
91     nebula  spine2-nxos  Ethernet1/25                                        absent             xcvr        
92     nebula  spine2-nxos  Ethernet1/26                                        absent             xcvr        
93     nebula  spine2-nxos  Ethernet1/27                                        absent             xcvr        
94     nebula  spine2-nxos  Ethernet1/28                                        absent             xcvr        
95     nebula  spine2-nxos  Ethernet1/29                                        absent             xcvr        
96     nebula  spine2-nxos  Ethernet1/30                                        absent             xcvr        
97     nebula  spine2-nxos  Ethernet1/31                                        absent             xcvr        
98     nebula  spine2-nxos  Ethernet1/32                                        absent             xcvr        
99     nebula  spine2-nxos  Ethernet1/33                                        absent             xcvr        
100    nebula  spine2-nxos  Ethernet1/34                                        absent             xcvr        
101    nebula  spine2-nxos  Ethernet1/35                                        absent             xcvr        
102    nebula  spine2-nxos  Ethernet1/36                                        absent             xcvr        
103    nebula  spine2-nxos  Ethernet1/37                                        absent             xcvr        
104    nebula  spine2-nxos  Ethernet1/38                                        absent             xcvr        
105    nebula  spine2-nxos  Ethernet1/39                                        absent             xcvr        
106    nebula  spine2-nxos  Ethernet1/40                                        absent             xcvr        
107    nebula  spine2-nxos  Ethernet1/41                                        absent             xcvr        
108    nebula  spine2-nxos  Ethernet1/42                                        absent             xcvr        
109    nebula  spine2-nxos  Ethernet1/43                                        absent             xcvr        
110    nebula  spine2-nxos  Ethernet1/44                                        absent             xcvr        
111    nebula  spine2-nxos  Ethernet1/45                                        absent             xcvr        
112    nebula  spine2-nxos  Ethernet1/46                                        absent             xcvr        
113    nebula  spine2-nxos  Ethernet1/47                                        absent             xcvr        
114    nebula  spine2-nxos  Ethernet1/48                                        absent             xcvr        
115    nebula  spine2-nxos  Ethernet1/49                                        absent             xcvr        
116    nebula  spine2-nxos  Ethernet1/50                                        absent             xcvr        
117    nebula  spine2-nxos  Ethernet1/51                                        absent             xcvr        
118    nebula  spine2-nxos  Ethernet1/52                                        absent             xcvr        
119    nebula  spine2-nxos  Ethernet1/53                                        absent             xcvr        
120    nebula  spine2-nxos  Ethernet1/54                                        absent             xcvr        
121    nebula  spine2-nxos  Ethernet1/55                                        absent             xcvr        
122    nebula  spine2-nxos  Ethernet1/56                                        absent             xcvr        
123    nebula  spine2-nxos  Ethernet1/57                                        absent             xcvr        
124    nebula  spine2-nxos  Ethernet1/58                                        absent             xcvr        
125    nebula  spine2-nxos  Ethernet1/59                                        absent             xcvr        
126    nebula  spine2-nxos  Ethernet1/60                                        absent             xcvr        
127    nebula  spine2-nxos  Ethernet1/61                                        absent             xcvr        
128    nebula  spine2-nxos  Ethernet1/62                                        absent             xcvr        
129    nebula  spine2-nxos  Ethernet1/63                                        absent             xcvr        
130    nebula  spine2-nxos  Ethernet1/64                                        absent             xcvr        
131    nebula  spine2-nxos       chassis                     Nexus9000 C9300v  present          chassis        
132    nebula  spine2-nxos    linecard-1  Nexus 9000v 64 port Ethernet Module  present         linecard        
133    nebula  spine2-nxos   linecard-27                    Supervisor Module  present         linecard
```

**Comment by rickdonato** [comment]
Just checked it does. Additional info added to the issue descr.

**Comment by ddutt** [comment]
Fixed via commit 06507652eac6888ea808cd574fb0c0f9906fc42e
